src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
You forgot to add `@Target` to the `@Inject` annotation. According to the checklist, it is necessary to specify the target for custom annotations.